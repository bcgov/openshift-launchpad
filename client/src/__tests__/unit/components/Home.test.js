import React from 'react';
import { shallow } from 'enzyme';

import Home from '@/components/Home';

describe('Home - unit', () => {
  
  it('renders correctly', () => {
    const wrapper = shallow(<Home />);
    expect(wrapper).toMatchSnapshot();
  });
  
});
