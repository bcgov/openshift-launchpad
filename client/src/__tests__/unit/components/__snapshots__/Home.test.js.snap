// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home - unit renders correctly 1`] = `ShallowWrapper {}`;
